src/sentry/api/endpoints/artifact_lookup.py

```diff
@@ -150,75 +152,41 @@ def get(self, request: Request, project: Project) -> Response:
         return Response(serialize(found_artifacts, request.user))
 
 
-def collect_artifact_bundles_containing_debug_ids(
-    debug_ids: List[str], project: Project, urls: List[str]
-) -> Tuple[Set[int], List[str]]:
+def get_artifact_bundle_containing_debug_id(debug_id: str, project: Project) -> Set[int]:
```

Comment:
Nit: I think this should be named in the plural.
```suggestion
def get_artifact_bundles_containing_debug_id(debug_id: str, project: Project) -> Set[int]:
```